docs/ProfileSchemaUse.md

```diff
@@ -20,4 +22,34 @@ to setup IntelliJ to validate json files against the schema follow these steps:
 1. select the `json` directory
 1. press okay
 
-now when you open a json file from the `json` directory in IntelliJ, it will be automatically validated against the DataHelix profile schema.
\ No newline at end of file
+now when you open a json file from the `json` directory in IntelliJ, it will be automatically validated against the DataHelix profile schema.
+
+
+## Microsoft Visual Studio Code
+
+to enable visual studio code to validate json files against the DataHelix profile schema a `json.schemas` section needs to be added to the `settings.json` file.
+
+to do this:
+
+1. click on the gear icon (<img src="../wikiimages/settingsicon.png" width="16" height="16">) at the bottom left of the screen and select `Settings`
+1. in the settings windows, click `Extensions` -> `JSON`
+1. you should see a section like this:
+    ```
+    Schemas
+    Associate schemas to JSON files in the current project
+    Edit in settings.json
+    ```
+1. click on the `Edit in settings.json` link and VSCode will open the settings.json file.
+1. add the following snippet to the end of the file (replacing `<datahelix_projectroot>` with the root directory path for the DataHelix project:
+    ```
+      "json.schemas": [
+        {
+          "fileMatch": [
+            "<datahelix_projectroot>/*"
+          ],
+          "url": "file://<datahelix_projectroot>/schemas/src/main/resources/profileschema/0.1/datahelix.schema.json"
```

Comment:
I had to use `file:/<datahelix_projectroot>` to get this to work

Reply:
my actual settings.json has :

```
"json.schemas": [
    {
      "fileMatch": [
        "/Users/aaspellc/src/datahelix/*"
      ],
      "url": "file://C:/Users/aaspellc/src/datahelix/schemas/src/main/resources/profileschema/0.1/datahelix.schema.json"
    }
```

the url file is `file://` same as `http://` or `ftp://`

so two slash's is correct

Reply:
okay, fixed the document. three slashes is actually correct !